var/spack/repos/builtin/packages/llvm/package.py

```diff
@@ -52,7 +52,7 @@ class Llvm(Package):
     depends_on('cmake @2.8.12.2:')
 
     # Universal dependency
-    depends_on('python@2.7:')
+    depends_on('python@2.7:2.8')  # Seems not to support python 3.X.Y
```

Comment:
`@2.7` should match any `2.7.x` and `@2` should match `2`.  Is this not the behavior you're seeing?  I'll check the tests.

I have wanted to put an exact match syntax in for a while, where `@2.7.` would match only `2.7` exactly (no additional specifiers) but I have not done that yet.